docker-compose.yml

```diff
@@ -7,6 +7,8 @@ services:
             dockerfile: server.Dockerfile
             context: ./requestor
         image: erigon-server
+        environment:
+            - SUBNET_TAG=${SUBNET_TAG:-erigon}
```

Comment:
This is bash magic to set default value. Otherwise (when no value provided) it's set to empty-string and fools `dict.get()` [here](https://github.com/golemfactory/yagna-service-erigon/blob/08c8ce5c51a3771932607399d71ad79c2af345ea/requestor/run_server.py#L8)

Reply:
:+1: